src/ember-app.js

```diff
@@ -450,6 +450,8 @@ function createShoebox(doc, fastbootInfo) {
 
     let value = shoebox[key];
     let textValue = JSON.stringify(value);
+    if (textValue === undefined) { continue; }
```

Comment:
Expected workflow is I'm jamming a bunch of keys and values in the shoebox on one side, then retrieving them on the other. I expect the values to be the exact same on the other side, whether undefined or not.

The alternative to this PR is to check against undefined every time I put something into the shoebox. This PR seems more developer friendly.